src/core_codemods/https_connection.py

```diff
@@ -24,7 +24,12 @@ def updated_args(self, original_args):
         return new_args
 
     def update_attribute(self, true_name, original_node, updated_node, new_args):
-        del true_name, original_node
+        del true_name
+        if not self.filter_by_path_includes_or_excludes(
+            self.node_position(original_node)
+        ):
+            return original_node
+
```

Comment:
The filtering call here is not needed as it is covered by the `leave_Call` in the parent class.